docs/content/docs/bundlers.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Fluid and Bundlers
+aliases:
+  - "/start/bundlers/"
+---
+
+The following are known bundler issues, answers to some of the most commonly asked questions about the Fluid
+Framework and Bundlers.
+
+## Known Errors with Webpack 5
+We know the following packages cause errors; assert, buffer, events and url.
+
+This issue causes compiling any project that uses these libraries to fail. Upgrading to Webpack 5 removed the automatic NodeJS polyfills. We now have to manually install the missing packages and add the fallback.
```

Comment:
JS is not a compiled language, so what does "compiling" mean? "bundling"? "transpiling from TypeScript"?

Reply:
"bundling" is the better term here.